tests/virtualization/universal/hotplugging.pm

```diff
@@ -100,9 +100,17 @@ sub add_virtual_disk {
     if (try_attach("virsh attach-disk --domain $_ --source $disk_image --target ${domblk_target}")) {
         assert_script_run "virsh domblklist $guest | grep ${domblk_target}";
         # Skip lsblk check for VIRT_AUTOTEST KVM test suites after attaching raw disk due to uncertainty
-        if (!get_var('VIRT_AUTOTEST') && is_kvm_host) {
-            my $lsblk = script_run("ssh root\@$guest lsblk | grep va[b-z]", 60);
-            record_soft_failure("lsblk failed - please check the output manually") if $lsblk != 0;
+        if (!get_var('VIRT_AUTOTEST')) {
+            if (is_kvm_host) {
+                my $lsblk = script_run("ssh root\@$guest lsblk | grep 'vd[b-z]'", 60);
+                record_soft_failure("lsblk failed - please check the output manually") if $lsblk != 0;
+            } elsif (is_xen_host) {
+                script_run("ssh root\@$guest lsblk");
```

Comment:
Suggested
`assert_script_run("ssh root\@$guest lsblk", fail_message => "lsblk failed - please check the output manually");`

Reply:
Thanks for the input! While this would look much better, it would also make the test fail instead of having a soft_failure, right?

Reply:
I would keep @grisu48 's approach as it uses to soft_failure. This is something we shouldfix later tho.